conmon/conmon.c

```diff
@@ -118,6 +119,7 @@ static GOptionEntry opt_entries[] =
 {
   { "terminal", 't', 0, G_OPTION_ARG_NONE, &opt_terminal, "Terminal", NULL },
   { "stdin", 'i', 0, G_OPTION_ARG_NONE, &opt_stdin, "Stdin", NULL },
+  { "leave-stdin-open", 0, 0, G_OPTION_ARG_NONE, &opt_leave_stdin_open, "do not close stdin even if remote sends EOF", NULL },
```

Comment:
should the first zero be a character?  Can you change this value without it?

Reply:
THis is a short option, I believe.  So specifying 0 means that no Short option is provided/allowed.